applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_3d.cpp

```diff
@@ -69,17 +69,27 @@ namespace Kratos
         const double& NU    = MaterialProperties[POISSON_RATIO];
 
         //NOTE: SINCE THE ELEMENT IS IN SMALL STRAINS WE CAN USE ANY STRAIN MEASURE. HERE EMPLOYING THE CAUCHY_GREEN
-        if(Options.IsNot( ConstitutiveLaw::COMPUTE_STRAIN )) //large strains
+        if(Options.Is( ConstitutiveLaw::COMPUTE_STRAIN ))
```

Comment:
to be honest i am not clear on the usage of this flag.

it may be that we need to change this agaon in the near future...